Formula/pygit2.rb

```diff
@@ -25,7 +26,7 @@ def python3
   end
 
   def install
-    system python3, *Language::Python.setup_install_args(prefix, python3)
+    system python3, "-m", "pip", "install", "--prefix=#{prefix}", "--no-deps", "."
```

Comment:
Just noting that I think this will install pre-built wheels for `build-system` dependencies. This case pulling latest `setuptools` and `wheel` binaries from PyPI.

One option would be to share the pip args that we use for venv DSL. The default args would still pull latest source packages (un-checksummed by Homebrew) but at least be built from source.

Alternatively for `setuptools`-only build where `python@3.11`'s copy works, then it can be done via `--no-build-isolation` (also an option in venv DSL). Not the case for more complex build-systems.

Reply:
ah oops, thanks will fix this. I know we don't currently checksum build-system deps but forgot this would pull wheels and not sdists.

I think this would be nice to move to a `std_pip_args` to match other languages, and then venv could share that. what do you think?